### PR TITLE
Re-export ActiveCamera in the root of amethyst_rendy

### PR DESCRIPTION
**Pull request by jojolepro**

### CODE REVIEW

**Comment by jojolepro** [comment]
wip

**Comment by jojolepro** [comment]
done

**Comment by Frizi** [comment]
rustfmt pass required

File: amethyst_rendy/src/camera.rs [hunk 1]
```diff
@@ -468,10 +468,10 @@ impl<'a> PrefabData<'a> for CameraPrefab {
 }
 
 /// Active camera prefab
-pub struct ActiveCameraPrefab(usize);
+pub struct ActiveCameraPrefab(Option<usize>);
```
Comment (torkleyy) [inline amethyst_rendy/src/camera.rs]: Is this even used anywhere? It doesn't even implement `Deserialize`, so I don't quite see how this is useful.

**Comment by jojolepro** [comment]
Fixed

@azriel91 the breaking change was that the Option was removed in the first place, and that it was not documented anywhere. This reverts the how it was before the rendy merge.

**Pull request merged**